pom.xml

```diff
@@ -955,7 +1128,7 @@
         <version>2.20.1</version>
         <configuration>
           <systemPropertyVariables>
-            <jacoco-agent.destfile>${project.build.directory}/jacoco/jacoco.exec</jacoco-agent.destfile>
+            <jacoco-agent.destfile>${project.build.directory}/jacoco.exec</jacoco-agent.destfile>
```

Comment:
why the change?

Reply:
just to use default path of jacoco plugin, it was my request.